src/keyDerivation_test.c

```diff
@@ -42,10 +46,10 @@ void testPrivateKeyDerivation()
 
 #define HD HARDENED_BIP32
 
-#define TESTCASE(path_, expected_) \
+#define TESTCASE(path_, expectedHex_) \
```

Comment:
I wonder whether we should switch underscore arguments to be normal and instead use different name inside the macro. Underscores might be easy to miss